addons/binding/org.openhab.binding.fritzaha/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: fritzAha Binding
+Bundle-SymbolicName: org.openhab.binding.fritzaha;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jackson-core-2.3.2.jar,
+ lib/jackson-databind-2.3.2.jar,
+ lib/jackson-module-jaxb-annotations-2.3.2.jar
+Import-Package: com.google.common.collect,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.openhab.io.net.http,
+ org.osgi.framework;version="1.8.0",
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.fritzaha,
+ org.openhab.binding.fritzaha.handler
+Require-Bundle: org.apache.commons.lang;bundle-version="2.6.0",
```

Comment:
please do not use require-bundle, but do import-package of all you need instead (and please without version constraint)